chain/chain-primitives/src/error.rs

```diff
@@ -180,7 +180,7 @@ pub enum ErrorKind {
     InvalidBalanceBurnt,
     /// Invalid shard id
     #[fail(display = "Shard id {} does not exist", _0)]
```

Comment:
maybe we should change this error message as well

Reply:
It's failing some test and I am trying to fix that now